packages/muelu/test/unit_tests/Adapters/AmgxOperatorAdapter.cpp

```diff
@@ -92,35 +92,35 @@ namespace MueLuTests {
       RCP<const Teuchos::Comm<int> > comm = TestHelpers::Parameters::getDefaultComm();
       int nx;
       //disable amgx test in parallel
-      if(comm->getSize() > 1) 
+      if(comm->getSize() > 1)
 	return;
       else
 	nx = 91;
 
       //matrix
-      RCP<Matrix> Op = TestHelpers::TestFactory<Scalar, LocalOrdinal, GlobalOrdinal, Node>::Build2DPoisson(nx, -1, Xpetra::UseTpetra); 
+      RCP<Matrix> Op = TestHelpers::TestFactory<Scalar, LocalOrdinal, GlobalOrdinal, Node>::Build2DPoisson(nx, -1, Xpetra::UseTpetra);
       RCP<Tpetra::CrsMatrix<Scalar,LocalOrdinal,GlobalOrdinal,Node> > tpA = MueLu::Utilities<Scalar, LocalOrdinal, GlobalOrdinal, Node>::Op2NonConstTpetraCrs(Op);
       RCP<Tpetra::Operator<Scalar,LocalOrdinal,GlobalOrdinal,Node> > tOp = tpA;
       Teuchos::ParameterList params, dummyList;
       params.set("use external multigrid package", "amgx");
       Teuchos::ParameterList subList = params.sublist("amgx:params", false);
       params.sublist("amgx:params").set("json file", "test.json");
-      RCP<MueLu::TpetraOperator<Scalar,LocalOrdinal,GlobalOrdinal,Node> > tH = MueLu::CreateTpetraPreconditioner(tOp, params,dummyList);
-      
+      RCP<MueLu::TpetraOperator<Scalar,LocalOrdinal,GlobalOrdinal,Node> > tH = MueLu::CreateTpetraPreconditioner(tOp, params);
+
       RCP<AMGXOperator> aH = Teuchos::rcp_dynamic_cast<AMGXOperator>(tH);
       TEST_EQUALITY(aH->sizeA()==nx*nx/comm->getSize(), true);
-   
+
       RCP<MultiVector> RHS = MultiVectorFactory::Build(Op->getRowMap(), 1);
       RCP<MultiVector> X   = MultiVectorFactory::Build(Op->getRowMap(), 1);
 
       //RHS=1, zero initial guess
       RHS->putScalar( (double) 1.0);
```

Comment:
You don't need to cast to `double` here.  Check out the following example: https://godbolt.org/z/x1huHU
```c++
#include <complex>
#include <type_traits>

int main () {
    std::complex<float> x ((double) 1.0, (double) 2.0);
    std::complex<float> y (1.0, 2.0);
    static_assert (std::is_same<decltype (1.0), double>::value, "uh oh");
    return 0;
}
```
Also, prefer C++ - style casts: either `double (1.0)` or `static_cast<double> (1.0)`.  (You don't need the cast here, but prefer these styles of cast in general.)